towhee/hub/repo_manager.py

```diff
@@ -275,6 +278,64 @@ def get_file_list(self, author: str, repo: str, commit: str) -> List[str]:
 
         return file_list
 
+    def download_executor(self, author: str, repo: str, tag: str, file_name: str, lfs_files: Tuple[str], local_dir: Union[str, Path]) -> bool:
```

Comment:
Why need `repo`? The `RepoManger` already has the repo info

Reply:
We might download other repo, for example we need to download pipeline-template when initialize a new pipeline repo.